--- /tmp/athen-scene-before-proxy.unity	2026-09-26 12:46:05.825115417 +0100
+++ unity/AthenHill/Assets/AthenHill/Scenes/AthenHill.unity	2026-09-26 12:45:51.431318040 +0100
@@ -13650,7 +13650,6 @@
   m_Children:
   - {fileID: 1206215599}
   - {fileID: 2422505734151740906}
-  - {fileID: 1049059654}
   m_Father: {fileID: 0}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 --- !u!114 &175973428
@@ -95447,7 +95446,7 @@
   m_LocalScale: {x: 0.42, y: 0.9, z: 0.42}
   m_ConstrainProportionsScale: 0
   m_Children: []
-  m_Father: {fileID: 175973427}
+  m_Father: {fileID: 2422505734151740906}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 --- !u!23 &1049059655
 MeshRenderer:
@@ -234443,7 +234442,10 @@
       objectReference: {fileID: 2100000, guid: 4da7200e630ff53619f7ac361d935f55, type: 2}
     m_RemovedComponents: []
     m_RemovedGameObjects: []
-    m_AddedGameObjects: []
+    m_AddedGameObjects:
+    - targetCorrespondingSourceObject: {fileID: 2936293656270133036, guid: de2c2efc341ab6235986b1c42bf3c1bd, type: 3}
+      insertIndex: -1
+      addedObject: {fileID: 1049059654}
     m_AddedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: de2c2efc341ab6235986b1c42bf3c1bd, type: 3}
 --- !u!4 &2422505734151740906 stripped
